Sources/OnLaunch/API/ApiActionLinkDto.swift

```diff
@@ -0,0 +1,37 @@
+struct ApiActionLinkDto: Decodable {
```

Comment:
RFC @damianjaeger if you agree with the naming of the constants

Reply:
Naming is fine